packages/market-service/src/marketDataCLI.ts

```diff
@@ -1,13 +1,16 @@
-// import { JsonRpcProvider } from '@ethersproject/providers'
-import { HistoryTimeframe } from '@shapeshiftoss/types'
+import dotenv from 'dotenv'
 
+// import { JsonRpcProvider } from '@ethersproject/providers'
+// import { HistoryTimeframe } from '@shapeshiftoss/types'
 // import { HistoryTimeframe } from '@shapeshiftoss/types'
 // import { Yearn } from '@yfi/sdk'
 // import { CoinGeckoMarketService } from './coingecko/coingecko'
 // import { YearnVaultMarketCapService } from './yearn/yearn-vaults'
 // import { YearnTokenMarketCapService } from './yearn/yearn-tokens'
 // import { OsmosisMarketService } from './osmosis/osmosis'
-import { FOXY_ASSET_ID, FoxyMarketService } from './foxy/foxy'
+// import { FOXY_ASSET_ID, FoxyMarketService } from './foxy/foxy'
+// import { MarketServiceManager } from './market-service-manager'
+dotenv.config()
```

Comment:
Ditto we shouldn't need this

Reply:
also, how useful is this file in general? would just remove unless we add a reasonable working default case or something...

Reply:
Yeah, looks like we're not really using the CLIs anymore at this point, maybe it be the market-data one or any of the CLI. The only time I remember using the CLI was during ATOM dev, when I used it for depositing / withdrawing, all of which could be done by e.g importing your seed in Keplr

Reply:
FWIW, I found the CLI to be extremely useful for quick testing in both this PR and #1176. 
At the same time, anything here can be done with Jest, and not having the CLI script available to use during development motivates writing and maintaining proper tests.